manifests/init.pp

```diff
@@ -58,8 +60,15 @@
     'Debian': {
       $package_name_default = 'cron'
       $service_name_default = 'cron'
+      $periodic_jobs_content_default = []
     }
     'Suse': {
+      $periodic_jobs_content_default = [
```

Comment:
I prefer arrays for better readability of the content. I respect people who prefer using strings in line 132 and it's tested in the specs (line 406). Maybe I should add a hint to the docs too.